projects/ngrx.io/content/guide/signals/rxjs-integration.md

```diff
@@ -15,7 +15,7 @@ import { map, pipe, tap } from 'rxjs';
 import { rxMethod } from '@ngrx/signals/rxjs-interop';
 
 @Component({ /* ... */ })
-export class NumbersComponent {
+export class Numbers {
```

Comment:
According to this comment from Matthieu, maybe a more specific name?

https://x.com/Jean__Meche/status/1925866491506831784

```suggestion
export class NumbersDoubler {
```